Userland/Libraries/LibWasm/WASI/Wasi.cpp

```diff
@@ -20,6 +20,11 @@
 #include <time.h>
 #include <unistd.h>
 
+#ifdef AK_OS_HAIKU
```

Comment:
Can we do this patch in AK/Platform.h? We have the relevant statements for BSD already

https://github.com/SerenityOS/serenity/blob/88a3b4da9e20c4a86cf6fbefb21e8a070d0731c9/AK/Platform.h#L228-L231